### PR TITLE
[android] Don't cut off top elements in navigation mode

### LINKED ISSUE #3732: Top elements are cut off in Android Navigation mode
That's a regression, likely related to #3654

Reproduces on Android 13 Pixel 6, and Android 9 Moto g6, not always, but 90% of the time.
It's happening for car, bike, and pedestrian navigation. No specific steps, on the road, with location following enabled (blue arrow button).

Entering & exiting the spit-screen or switching apps doesn't solve the issue. The top is still cut off.

### PR DESCRIPTION
**Pull request by rtsisyk**
Fixes #3732 

Signed-off-by: Roman Tsisyk <roman@tsisyk.com>

###

image

### CODE REVIEW

**Comment by arnaudvergnet** [comment]
Oh yes nice catch! When testing the street name was always shown and I constrained the next turn instruction to the street name panel to make sure they move at the same time. But yeah if you set the street visibility to none it will definitely break the layout.

**Review by arnaudvergnet — APPROVED** [review]

File: android/src/app/organicmaps/routing/NavigationController.java [hunk 1]
```diff
@@ -214,10 +214,10 @@ public void update(@Nullable RoutingInfo info)
 
   private void updateStreetView(@NonNull RoutingInfo info)
   {
-    boolean hasStreet = !TextUtils.isEmpty(info.nextStreet);
-    UiUtils.showIf(hasStreet, mStreetFrame);
-    if (!TextUtils.isEmpty(info.nextStreet))
-      mNextStreet.setText(info.nextStreet);
+    mNextStreet.setText(info.nextStreet);
```
Comment (vng) [inline android/src/app/organicmaps/routing/NavigationController.java:217]: If the fix is about showId -> visibleIf, should we keep setText under if not isEmpty?

Comment (rtsisyk) [reply android/src/app/organicmaps/routing/NavigationController.java:217]: Yes, we can. I just wanted to streamline the code.

Comment (rtsisyk) [reply android/src/app/organicmaps/routing/NavigationController.java:217]: @vng keep this refactored version or minimize the patch?

Comment (vng) [reply android/src/app/organicmaps/routing/NavigationController.java:217]: Hm, better to leave under if, I think. What if info.nextStreet is null?

Comment (rtsisyk) [reply android/src/app/organicmaps/routing/NavigationController.java]: ok, no brainer.

**Pull request merged**